syntax_and_semantics/modules.md

```diff
@@ -123,3 +123,60 @@ end
 
 Moo.new # undefined method 'new' for Moo:Module
 ```
+
+# Module Type Checking
+
+Modules can also be used for type checking.
+
+If we define two modules with names `A` and `B`:
+
+```crystal
+module A; end
+module B; end
+```
+
+These can be included into classes:
+
+```crystal
+class One
+  include A
+end
+
+class Two
+  include B
+end
+
+class Three
+  include A
+  include B
```

Comment:
Maybe show transitivity by inheriting from `Two` instead of including `B`?

Reply:
Done